examples/08_inplace_FFT/08_inplace_FFT.cpp

```diff
@@ -0,0 +1,88 @@
+// SPDX-FileCopyrightText: (C) The kokkos-fft development team, see COPYRIGHT.md file
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
+
+#include <Kokkos_Core.hpp>
+#include <Kokkos_Complex.hpp>
+#include <Kokkos_Random.hpp>
+#include <KokkosFFT.hpp>
+
+using execution_space = Kokkos::DefaultExecutionSpace;
+
+template <typename T>
+using RightView2D = Kokkos::View<T **, Kokkos::LayoutRight, execution_space>;
+
+template <std::size_t DIM>
+using axis_type = KokkosFFT::axis_type<DIM>;
+template <std::size_t DIM>
+using shape_type = KokkosFFT::shape_type<DIM>;
+
+int main(int argc, char *argv[]) {
+  Kokkos::initialize(argc, argv);
+  {
+    constexpr int n0 = 128, n1 = 128;
+    const Kokkos::complex<double> z(1.0, 1.0);
+
+    shape_type<2> shape;
+    shape[0] = n0;
+    shape[1] = n1;
```

Comment:
Can we initialize `shape{n0, n1}` ?

Reply:
True. I will fix that

Reply:
Actually, this variable is unnecessary. I have removed this

Reply:
This was not detected by the warnings as errors ?